drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -4812,7 +4812,8 @@ Vector<uint8_t> RenderingDeviceVulkan::shader_compile_binary_from_spirv(const Ve
 		offset += sizeof(uint32_t);
 		memcpy(binptr + offset, &binary_data, sizeof(RenderingDeviceVulkanShaderBinaryData));
 		offset += sizeof(RenderingDeviceVulkanShaderBinaryData);
-		memcpy(binptr + offset, shader_name_utf.ptr(), binary_data.shader_name_len);
+		if (shader_name_utf.ptr() != nullptr)
+			memcpy(binptr + offset, shader_name_utf.ptr(), binary_data.shader_name_len);
```

Comment:
```suggestion
		if (shader_name_utf.ptr() != nullptr) {
			memcpy(binptr + offset, shader_name_utf.ptr(), binary_data.shader_name_len);
		}
```
As per the code style